src/locales/en.json

```diff
@@ -218,6 +221,10 @@
       ]
     }
   },
+  "profilePage": {
+    "title": "Welcome back"
```

Comment:
What if user logs in for the first time?

Reply:
PTAL